simtools/applications/production_generate_simulation_config.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/python3
+
+r"""
+Configure a simulation based on command-line arguments.
+
+This application configures and
+generates simulation parameters for a specific grid point in a statistical uncertainty
+evaluation setup.
+
+Command line arguments
+----------------------
+azimuth (float, required)
+    Azimuth angle in degrees.
+elevation (float, required)
+    Elevation angle in degrees.
+nsb (float, required)
+    Night sky background value.
+ctao_data_level (str, required)
+    The data level for the simulation (e.g., 'A', 'B', 'C').
+science_case (str, required)
+    The science case for the simulation.
+file_path (str, required)
+    Path to file with MC events at CTAO DL2 data level. Used for statistical uncertainty evaluation.
+file_type (str, required)
+    Type of the dl2_mc_events_file file ('point-like' or 'cone').
+metrics (str, optional)
+    Path to a YAML file containing metrics for evaluation.
+site (str, required)
+    The observatory site (North or South).
+
+Example
+-------
+To run the simulation configuration, execute the script as follows:
+
+.. code-block:: console
+
+    simtools-production-generate-simulation-config --azimuth 60.0 --elevation 45.0 \
+      --nsb 0.3 --ctao_data_level "A" --science_case "high_precision" \
+      --file_path "path/to/fits_file.fits" --file_type "On-source" \
```

Comment:
`On-source` as file type, while the docstring above lists other options (point-like/cone)